contents/blog/why-redesign.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Why are we redesigning PostHog?
+date: 2023-11-27
+author: ["cory-watilo"]
+rootpage: /blog
+featuredImage: ../images/blog/why-redesign/preview-3000.jpg
+featuredImageType: full
+category: PostHog news
+tags:
+  - Product updates
+---
+
+A year ago, [James](/james) and I sat down in a coffee shop in Austin, Texas and had a discussion about what the future of PostHog could look like.
+
+Our current UI has served its purpose, but as we grow as a company, the possibilities of what can be achieved in PostHog have grown – as have our ambitions.
```

Comment:
```suggestion
We realized that while our current UI has served its purpose, the possibilities of what you can do in PostHog have grown – as have our ambitions.
```

Make the user the main focus of the story, remove passive voice, tie the framing to the Austin conversation (because if you don't, it's not relevant).